generator/wrapper_gen.py

```diff
@@ -181,7 +181,7 @@ def _get_data(x):
         else:
             x = [xi.to_numpy() for _, xi in x.items()]
     elif isinstance(x, pdSeries):
-        x = [x.to_numpy()]
+        x = x.to_numpy()
```

Comment:
Hm, does this return a 1d or 2d array?

Reply:
1d array:

```
In [60]: ps = pd.Series(X[:, 0])

In [61]: X[:,0].shape
Out[61]: (10,)

In [62]: ps.to_numpy().shape
Out[62]: (10,)
```

Reply:
Thanks, fixed! Also added tests for this case.